### PR TITLE
new emqx ee 44 test

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: ansible/roles/emqx5/tasks/configure.yml [hunk 1]
```diff
@@ -70,3 +70,17 @@
...
+    src: "/etc/ssl/certs/emqx/{{ item }}"
+    dest: "/etc/emqx/certs/"
+    remote_src: yes
+    mode: '0644'
+  with_items:
+    - "cacert.pem"
+    - "cert.pem"
+    - "key.pem"
+    - "client-cert.pem"
+    - "client-key.pem"
```
Comment (thalesmg) [inline ansible/roles/emqx5/tasks/configure.yml:86]: Q: should the server and client bundles also be copied?

File: ansible/roles/emqx4/tasks/install-package.yml [hunk 2]
```diff
@@ -45,6 +45,20 @@
...
+    src: "/etc/ssl/certs/emqx/{{ item }}"
+    dest: "/etc/emqx/certs/"
+    remote_src: yes
+    mode: '0644'
+  with_items:
+    - "cacert.pem"
+    - "cert.pem"
+    - "key.pem"
+    - "client-cert.pem"
+    - "client-key.pem"
```
Comment (thalesmg) [inline ansible/roles/emqx4/tasks/install-package.yml:60]: Should copy server and client bundles?

File: main.tf [hunk 3]
```diff
@@ -232,273 +236,11 @@ resource "aws_lb_target_group_attachment" "prometheus" {
...
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/node_exporter.yml"
-    environment = {
-      no_proxy = "*"
-    }
+module "certs" {
+  source = "./modules/certs"
```
Comment (thalesmg) [inline main.tf:240]: This seems to be missing from the repo/branch?

File: ansible/roles/emqx5/tasks/configure.yml (hunk 1 above)
Comment (id) [reply ansible/roles/emqx5/tasks/configure.yml:86]: we can do it, but emqx does not use bundles

File: main.tf (hunk 3 above)
Comment (id) [reply main.tf:240]: oops, fixed

**Review by thalesmg — APPROVED** [review]

**Pull request merged**